dev/CommonStyles/ContentDialog_themeresources.xaml

```diff
@@ -0,0 +1,16 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+
+    <ResourceDictionary.ThemeDictionaries>
+        <ResourceDictionary x:Key="Default">
+            <SolidColorBrush x:Key="SystemControlPageBackgroundMediumAltMediumBrush" Color="#1E000000" />
```

Comment:
I think we want to change the brush that is used by ContentDialog dismiss layer, not the value of that brush itself since the brush might be used by other controls too, changing the value directly will affect all of them.

Reply:
We should bring in ContentDialog template and then modify the Background property of dismiss layer to use the right brush.

Checkout the script at `tools\ExtractControls.ps1`, that will help you bring over the template and theme resources from OS.